src/app.coffee

```diff
@@ -36,15 +39,21 @@ defaults =
   driver: "memory"
   params: {}
   address: null,
-  
-if fs.existsSync(CONFIG_FILE)
-  try
-    config = JSON.parse(fs.readFileSync(CONFIG_FILE))
-  catch err
-    console.error err
-    process.exit 1
-else
-  config = {}
+  port: null
+
+if process.env.HOME
+  CONFIG_FILES.push path.join process.env.HOME, ".ofirehose.json"
+
+getConfig = (file) ->
+  if fs.existsSync(file)
```

Comment:
Instead of checking if the file exists, you should attempt a read and then handle the error if it's not there. See https://nodejs.org/api/fs.html#fs_fs_access_path_mode_callback (just do a Find for "race condition")

Reply:
Yes, and existSync is deprecated